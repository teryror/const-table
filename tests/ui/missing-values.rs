use const_table::const_table;

#[const_table]
enum Foo {
    Bar { bazz: i32 }
}

fn main() {}