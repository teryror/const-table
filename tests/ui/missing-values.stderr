error: a const_table enum needs at least one variant with a discriminant expression
 --> $DIR/missing-values.rs:4:10
  |
4 |   enum Foo {
  |  __________^
5 | |     Bar { bazz: i32 }
6 | | }
  | |_^
